### PR TITLE
[BE] Fix SIM109 `compare-with-tuple`

### PR DESCRIPTION
**Pull request by justinchuby**
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #100336
* #100342
* #100341
* #100339
* #100338
* __->__ #100337
* #100335

Use {replacement} instead of multiple equality comparisons

### CODE REVIEW

**Review by Skylion007 — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -48,7 +48,6 @@ ignore = [
     "SIM102", "SIM103", "SIM112", # flake8-simplify code styles
     # these ignores are from flake8-simplify. please fix or ignore with commented reason
     "SIM108",
-    "SIM109",
```
Comment (Skylion007) [inline pyproject.toml:51]: Also please edit the .flake8 config to remove the ignore from there as well.

Comment (justinchuby) [reply pyproject.toml:51]: Done

**Pull request closed without merging**